CHANGELOG.md

```diff
@@ -7,6 +7,27 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [5.4.0] - 2019-05-20
+
+### Added
+- New AsyncAPI 2 ruleset and format [#965](https://github.com/stoplightio/spectral/issues/965)
+- Custom functions can be async [#694](https://github.com/stoplightio/spectral/issues/694)
+- Expose fetch and cache to custom functions [#1078](https://github.com/stoplightio/spectral/pull/1078), [#1079](https://github.com/stoplightio/spectral/pull/1079)
+
+### Changed
+- Rename `oas3-example-value-or-externalValue` rule to `oas3-examples-value-or-externalValue` [#1098](https://github.com/stoplightio/spectral/pull/1098)
+- Drop useless oas rule tags [#1095](https://github.com/stoplightio/spectral/pull/1095)
+- AJV: swallow missing references [#1147](https://github.com/stoplightio/spectral/pull/1147)
+- Improve ruleset validation [#1144](https://github.com/stoplightio/spectral/pull/1144)
+- Allow running spectral from any directory in Docker [#1022](https://github.com/stoplightio/spectral/pull/1022)
+
+### Fixed
+- Improve `oas2-schema` and `oas3-schema` rules [#403](https://github.com/stoplightio/spectral/issues/403)
+- Make `example-value-or-externalValue` only target oas3 document [#883](https://github.com/stoplightio/spectral/issues/883)
```

Comment:
I'd drop this from here and update the `Changed` section to 

```
Drop `example-value-or-externalValue` rule and introduce `oas3-examples-value-or-externalValue` [#883](https://github.com/stoplightio/spectral/issues/883) and [#1098](https://github.com/stoplightio/spectral/pull/1098)
```

From the overall release standpoint, we haven't fixed `example-value-or-externalValue`, we've nuked it ;-)

Reply:
[keep a changelog](https://keepachangelog.com/en/1.0.0/) has the 'removed' section.
Seems like a perfect fit here.